Gruntfile.js

```diff
@@ -143,7 +147,7 @@ module.exports = function( grunt ) {
           cwd: 'svg-min-react/',
           src: [ '**/*.svg' ],
           filter: 'isFile',
-          dest: 'build/'
+          dest: 'dist/'
```

Comment:
I'm fine with moving the (CJS) stuff we're going to publish to `dist/`; however, what about still building the intermediary ES6 files (that, IIUC, we aren't publishing for now) in `build/`? Just to keep things a bit more separate -- the `dist/` dir will then only hold stuff that we're also going to publish. That will allow us to set `"files": [ "dist/" ]` instead of `"files": [ "dist/*js" ]` in `package.json`. (IMO, the latter is both uglier, and more fragile, since we're relying on `*.js` (CJS) vs `*.jsx` (ES6) extensions, which is quite subtle and easy to miss.)